analyses/cms-open-data-ttbar/ttbar_analysis_pipeline.py

```diff
@@ -150,6 +141,7 @@ def __init__(self, disable_processing, io_branches):
         )
         self.disable_processing = disable_processing
         self.io_branches = io_branches
+        self.cset = correctionlib.CorrectionSet.from_file("corrections.json")
```

Comment:
I did not think about this before: this will not work with Dask unless we also ship the file around, right? At the moment this would not get picked up. I imagine conceptually we could also build the `CorrectionSet` outside of the processor and hand it to `__init__` ready to go? Not sure if the pickling will automatically make that work though.

Reply:
Commenting here for documentation purposes. I think this does end up working with Dask, but only b/c the `TtbarAnalysis` class is defined within the jupyter notebook itself. If we instead had an `import` call to get the class, the Dask workers would indeed complain about not having the `corrections.json` file. I haven't explicitly tested this though.

Reply:
Indeed, Dask looks to be working just fine with this current implementation.